packages/models/src/lib/util/initial-results.model.ts

```diff
@@ -0,0 +1,8 @@
+import { ContentModel } from '../content';
+import { User } from '../users';
+
+export interface InitialResults {
+    users: User[],
+    blogs: ContentModel[],
+    works: ContentModel[],    
```

Comment:
This is one bit of weirdness--the initial implementation for searching just hits the content service, so it just gets back ContentModels. I could've implemented searches down in the individual services, but it doesn't look like we need that yet. (if we do in the future, we can always implement it there too)